src/Microsoft.VisualStudio.ProjectSystem.Managed/StringComparers.cs

```diff
@@ -29,5 +29,10 @@ public static IEqualityComparer<string> ConfigurationDimensionNames
         {
             get { return StringComparer.Ordinal; }
         }
+
+        public static IEqualityComparer<string> DependencyProviderTypes
+        {
+            get { return StringComparer.OrdinalIgnoreCase; }
```

Comment:
Actually, it's inconsistent across the team, I'll unify it and make it Ordinal in a future change.